synapse/res/templates/sso_redirect_confirm.html

```diff
@@ -2,35 +2,50 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>SSO redirect confirmation</title>
+    <title>Your account is now ready</title>
     <meta name="viewport" content="width=device-width, user-scalable=no">
+    <link rel="stylesheet" type="text/css" href="sso.css">
```

Comment:
@bwindels there is no sso.css?

Reply:
[there is](https://github.com/matrix-org/synapse/blob/develop/synapse/res/templates/sso.css)?

Reply:
ahh, this is a leftover from local development. Please remove this.

Reply:
done in 06d8de23b6